### PR TITLE
Update python-mode snippet

### PR DESCRIPTION
**Pull request by zhenzong**

### CODE REVIEW

File: python-mode/str [hunk 1]
```diff
@@ -4,4 +4,6 @@
 # group: dunder methods
 # --
 def __str__(self):
-    $0
\ No newline at end of file
+    $0
+
+__repr__ = __str__
```
Comment (AndreaCrotti) [inline python-mode/str]: mm not sure I agree with this one, repr is often not the same as str..

File: python-mode/doc [hunk 2]
```diff
@@ -2,5 +2,6 @@
 # name: doc
 # key: d
 # --
-"""$0
+"""
+$0
```
Comment (AndreaCrotti) [inline python-mode/doc]: This also is not necessarily always the right thing to do I think..

File: python-mode/str (hunk 1 above)
Comment (zhenzong) [inline python-mode/str]: Ok, I will discard this modify. Thank for reply.

File: python-mode/doc (hunk 2 above)
Comment (zhenzong) [inline python-mode/doc]: Your are right. I check [pep](https://www.python.org/dev/peps/pep-0257/) again, my fault.

**Comment by AndreaCrotti** [comment]
Ok great thanks

**Pull request merged**